pkg/controller/podgc/gc_controller.go

```diff
@@ -179,11 +179,11 @@ func (gcc *PodGCController) gcTerminating(ctx context.Context, pods []*v1.Pod) {
 		wait.Add(1)
 		go func(pod *v1.Pod) {
 			defer wait.Done()
-			deletingPodsTotal.WithLabelValues().Inc()
+			deletingPodsTotal.WithLabelValues("out-of-service").Inc()
```

Comment:
Can we move these dimensions such as `out-of-service`, `terminated` etc in well defined constants, so as it gets easier to reuse them without potentially having typos in the code?

Reply:
@gnufied updated.